hyp3lib/get_asf.py

```diff
@@ -1076,69 +1075,55 @@ def signal_handler(signum, frame):
+            cfg.new.update(g1)
+        download_granules(cfg)
 
-            log.debug("Platform(s): " + str(cfg.platforms))
-            log.debug("Beam Mode(s): " + str(cfg.beammodes))
-            log.debug("WKT: " + str(cfg.wkt))
-            log.debug("Point: " + str(cfg.point))
-            log.debug("Level: " + str(cfg.level))
-            log.debug("Dry Run: " + str(cfg.dry_run))
+    else:
+        start = None
+        if cfg.start_time:
+            if "T" in cfg.start_time:
+                start = datetime.datetime.strptime(cfg.start_time, "%Y-%m-%dT%H:%M:%S")
+            else:
+                start = datetime.datetime.strptime(cfg.start_time, "%Y-%m-%d")
+            log.debug("Start Time: " + str(start))
 
-            cfg.new = find_granules_search(cfg.platforms, cfg.beammodes, start, end, cfg.wkt, cfg.point, cfg.max,
-                                           cfg.level, wget_options=cfg.wget_options)
+        end = None
+        if cfg.end_time:
+            if "T" in cfg.end_time:
+                end = datetime.datetime.strptime(cfg.end_time, "%Y-%m-%dT%H:%M:%S")
+            else:
+                end = datetime.datetime.strptime(cfg.end_time, "%Y-%m-%d")
+            log.debug("End Time: " + str(start))
 
-            #g = find_granules('ALOS', 'FBS', datetime.datetime.strptime("2008-12-01T00:00:00", "%Y-%m-%dT%H:%M:%S"),
-            #                  datetime.datetime.strptime("2008-12-31T00:00:00", "%Y-%m-%dT%H:%M:%S"),
-            #                  "-155.08,65.82,-153.28,64.47,-149.94,64.55,-149.50,63.07,-153.5,61.91")
-            download_granules(cfg)
+        log.debug("Platform(s): " + str(cfg.platforms))
+        log.debug("Beam Mode(s): " + str(cfg.beammodes))
+        log.debug("WKT: " + str(cfg.wkt))
+        log.debug("Point: " + str(cfg.point))
+        log.debug("Level: " + str(cfg.level))
+        log.debug("Dry Run: " + str(cfg.dry_run))
 
-        log.info("Done")
+        cfg.new = find_granules_search(cfg.platforms, cfg.beammodes, start, end, cfg.wkt, cfg.point, cfg.max,
+                                       cfg.level, wget_options=cfg.wget_options)
 
-    # if we get a SystemExit from down in, pass it thru silently
-    except SystemExit:
-        raise
-    # trap Ctrl-C
-    except KeyboardInterrupt:
-        log.critical("Keyboard interrupt received (Ctrl-C).")
-    except Exception:
-        log.critical("Uncaught exception seen, traceback follows.")
-        tb = traceback.format_exc().split("\n")
-        for tbline in tb:
-            log.info(tbline)
-        log.critical("Aborting due to uncaught exception.")
+        #g = find_granules('ALOS', 'FBS', datetime.datetime.strptime("2008-12-01T00:00:00", "%Y-%m-%dT%H:%M:%S"),
+        #                  datetime.datetime.strptime("2008-12-31T00:00:00", "%Y-%m-%dT%H:%M:%S"),
+        #                  "-155.08,65.82,-153.28,64.47,-149.94,64.55,-149.50,63.07,-153.5,61.91")
+        download_granules(cfg)
+
+        log.info("Done")
```

Comment:
looks like this changes the exit status, I assume that's intentional?  Previously `KeyboardInterrupt` and `Exception` would have exited quietly, now they'll exit noisily?

Reply:
Yeah, for both they will exit with a non-zero status now, and write output to `stderr` instead of `stdout`. That may have been intentional at some time, but it's bad practice IMO. 

This change will cause `execute('get_asf.py ...')` commands to fail, which is how we download granules, if granules couldn't be downloaded, instead of silently failing and pretending it worked. I supposed that's a substantial change and could be reverted b/c of it. 

The `KeyboardInterupt` failing quietly I suppose could be desired by users, I guess. This is what it'd look like now: 
![image](https://user-images.githubusercontent.com/7882693/84083378-f985cd80-a98d-11ea-9900-9744bdcfc739.png)

![image](https://user-images.githubusercontent.com/7882693/84083333-e5da6700-a98d-11ea-8293-c51c4dda4d9a.png)